.gitmodules

```diff
@@ -2,6 +2,8 @@
   path = spack
   url = https://github.com/jcsda/spack
   branch = spack-stack-dev
+  url = https://github.com/climbfuji/spack
```

Comment:
Is this okay: two branch and url specifications?

Reply:
Ooops. Fixed after merging the spack PR and reverting the `.gitmodules` changes.